Formula/poac.rb

```diff
@@ -0,0 +1,29 @@
+class Poac < Formula
+  desc "Package manager for C++"
+  homepage "https://github.com/poacpm"
+  url "https://github.com/poacpm/poac.git",
+      :tag      => "0.1.2",
+      :revision => "2a38827050cd797827c8b6154240454ffc518d53"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on :macos => :sierra
+  depends_on "openssl"
+  depends_on "yaml-cpp"
+
+  def install
+    mkdir "bulid" do
+      system "cmake", "..", *std_cmake_args
```

Comment:
Somehow it looks like it is hardcoding the compiler path?

```
/usr/local/opt/llvm/bin/clang++  -DPOAC_PROJECT_ROOT=\"/tmp/poac-20190211-22098-1vavzz3\" -DPOAC_VERSION=\"0.1.2\" -I/usr/local/include -I/usr/local/opt/openssl/include  -std=c++17 -pthread -I/usr/local/opt/llvm/include -DNDEBUG   -fdiagnostics-color -Wall -Wextra -Werror -o CMakeFiles/poac.dir/main.cpp.o -c /tmp/poac-20190211-22098-1vavzz3/main.cpp
make[2]: /usr/local/opt/llvm/bin/clang++: No such file or directory
make[2]: *** [CMakeFiles/poac.dir/main.cpp.o] Error 1
```

Reply:
I get the following error in case of no hardcoding the compiler path.
```
Call to unavailable function 'visit': introduced in macOS 10.14
```
Therefore, if macOS is not 10.14, able to build by using to llvm.